DataService/schema.json

```diff
@@ -1,28 +1,128 @@
+        "accessRights": {
+          "type": "string",
+          "description": "Property. Model:'http://purl.org/dc/terms/RightsStatement'. This property MAY include information regarding access or restrictions based on privacy, security, or other policies."
+        },
+        "applicableLegislation": {
+          "type": "array",
+          "description": "Property. Model:'http://data.europa.eu/r5r/applicableLegislation'. The legislation that mandates the creation or management of the Data Service. Cardinality: 0..n.",
+          "items": {
+            "type": "string",
+            "description": "Property. Model:'http://data.europa.eu/eli/ontology#LegalResource'. This class represents the legislation, policy or policies that lie behind the Rules that govern the service. The definition and properties of the Legal Resource class are aligned with the ontology included in \"Council conclusions inviting the introduction of the European Legislation Identifier (ELI - https://eur-lex.europa.eu/eli-register/about.html)\". For describing the attributes of a Legal Resource (labels, preferred labels, alternative labels, definition, etc.) we refer to the (ELI - https://op.europa.eu/en/web/eu-vocabularies/eli) ontology. In this data specification the use is restricted to instances of this class that follow the (ELI - https://op.europa.eu/en/web/eu-vocabularies/eli) URI guidelines."
+          }
+        },
+        "conformsTo": {
+          "type": "array",
+          "description": "Property. Model:'http://purl.org/dc/terms/conformsTo'. An established (technical) standard to which the Data Service conforms. The standards referred here SHOULD describe the Data Service and not the data it serves. The latter is provided by the dataset with which this Data Service is connected. For instance the data service adheres to the OGC WFS API standard, while the associated dataset adheres to the INSPIRE (https://knowledge-base.inspire.ec.europa.eu/index_en) Address data model. Cardinality: 0..n.",
+          "items": {
+            "type": "string",
+            "description": "Property. Model:'https://semiceu.github.io/DCAT-AP/releases/3.0.0/#Standard'. A standard or other specification to which a resource conforms. This specification does not impose any additional requirements to properties for this entity."
+          }
         },
-        "endPointURL": {
+        "contactPoint": {
+          "type": "array",
+          "description": "Property. Model:'http://www.w3.org/ns/dcat#contactPoint'. Contact information that can be used for sending comments about the Data Service. Cardinality: 0..n.",
+          "items": {
+            "type": "object",
+            "description": "Property. Model:'https://www.w3.org/ns/dcat#contactPoint'. Relevant contact information for the cataloged resource. Use of vCard is recommended (https://www.w3.org/TR/vcard-rdf/#d4e1819)."
```

Comment:
Where are the properties of this object defined?

Reply:
"properties": {
  "email": {
    "type": "string",
    "description": "Property. To specify the electronic mail address for communication with the object the vCard represents"
   "format": "email"
   }
}}